site/_posts/2017-05-16-lwimiq-words.md

```diff
@@ -0,0 +1,113 @@
+(
+[ManageIQ/manageiq#14923](https://github.com/ManageIQ/manageiq/pull/14923)
+)
+
+Libor made Middleware models and MiqRequest work with RBAC.
+(
+[ManageIQ/manageiq#15011](https://github.com/ManageIQ/manageiq/pull/15011),
+[ManageIQ/manageiq-ui-classic#1214](https://github.com/ManageIQ/manageiq-ui-classic/pull/1214)
+)
+
+
+## UI side
+
+On the UI (classic) side [34 PRs](https://github.com/ManageIQ/manageiq-ui-classic/pulls?q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222017-05-08+..+2017-05-14%22+sort%3Acreated-desc&utf8=%E2%9C%93) were merged last week.
+
+Thanks to Yaacov, Containers now no longer use an explorer screen (the one with a tree on the left) and replaced it with a listnav, adding some default filters in the process.
+(
+[ManageIQ/manageiq#14893](https://github.com/ManageIQ/manageiq/pull/14893),
+[ManageIQ/manageiq-ui-classic#1204](https://github.com/ManageIQ/manageiq-ui-classic/pull/1204)
+)
+
+Speaking of listnavs, Gabriel also added a listnav to the Physical Server screen, for the new physical infrastructure.
+Not stopping there, he also added a textual summary link to those, list of groups and a toolbar.
+(
+[ManageIQ/manageiq-ui-classic#1195](https://github.com/ManageIQ/manageiq-ui-classic/pull/1195),
+[ManageIQ/manageiq-ui-classic#1322](https://github.com/ManageIQ/manageiq-ui-classic/pull/1322),
+[ManageIQ/manageiq-ui-classic#1299](https://github.com/ManageIQ/manageiq-ui-classic/pull/1299),
+[ManageIQ/manageiq-ui-classic#1294](https://github.com/ManageIQ/manageiq-ui-classic/pull/1294)
+)
+
+Physical servers were rather popular last week it seems, since Maysa was also working there, adding a hardware info display in the detail screen, and Charlie has added network data to the same screen.
+(
+[ManageIQ/manageiq-ui-classic#1272](https://github.com/ManageIQ/manageiq-ui-classic/pull/1272),
+[ManageIQ/manageiq-ui-classic#1276](https://github.com/ManageIQ/manageiq-ui-classic/pull/1276)
+)
+
+Aparna fixed Ansible credentials password handling when editing.
+(
+[ManageIQ/manageiq-ui-classic#1227](https://github.com/ManageIQ/manageiq-ui-classic/pull/1227)
+)
+
+Eric made PDFs work again after the UI split.
+(
+[ManageIQ/manageiq-ui-classic#1332](https://github.com/ManageIQ/manageiq-ui-classic/pull/1332),
+[ManageIQ/manageiq#14793](https://github.com/ManageIQ/manageiq/pull/14793)
+)
+
+Zita made it possible for container dashboards to display flash messages.
+(
+[ManageIQ/manageiq-ui-classic#1312](https://github.com/ManageIQ/manageiq-ui-classic/pull/1312)
+)
+
+And Martin did a toolbar cleanup, we no longer have to show/hide the toolbar manually from the UI code, the toolbar now autohides when no toolbars are present, and shows up when some are.
+(
+[ManageIQ/manageiq-ui-classic#1239](https://github.com/ManageIQ/manageiq-ui-classic/pull/1239)
+)
+
+We also had a bunch of Mendel University in Brno students convert some Angular controllers to `controllerAs`, as part of their Software Engineering course, thanks! :)
```

Comment:
@juliancheal another dyslexic reader here! Nonetheless, perhaps "students from Mendel University in Brno" might help avoid reading this as "a bunch of Mendel University in Brno" as I also did at first, even though this is techinically correct!